### DISCUSSION TITLE
Issue with the precision of quality flag

### DISCUSSION THREAD

**Discussion by diesseh** [maawoo/gedixr#17] [Q&A] [answered]
I am getting this error

>>> gdf_l2a = extract_data(directory=gedi_dir, gedi_product='L2A', only_full_power=True, subset_vector=vec_path)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: extract_data() got an unexpected keyword argument 'only_full_power'

**Comment by maawoo** [comment, marked as answer]
Hi @diesseh, 
see my answer in the issue you opened: https://github.com/maawoo/gedixr/issues/18
